GpxExport/ViewControllers/WorkoutsTableViewController.swift

```diff
@@ -90,6 +90,19 @@ class WorkoutsTableViewController: UITableViewController {
     }
 
     private func exportSelectedWorkouts() {
+        let alert = UIAlertController(title: "What file to generate?", message: "It's recommended to use GPX for now.", preferredStyle: .actionSheet)
+
+        alert.addAction(UIAlertAction(title: "GPX", style: .default, handler: {(_: UIAlertAction!) in
```

Comment:
you cloud use the handler without force unwrapping.

Reply:
Oops. Maybe an copy/paste issue